test/support/tc.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require "testcontainers/mysql"
+
+module TC
+  class << self
+    def start_mysql
+      @pool_1_container = Testcontainers::MysqlContainer.new("mysql:8.0", username: "root", password: "")
+      @pool_2_container = Testcontainers::MysqlContainer.new("mysql:8.0", username: "root", password: "")
```

Comment:
Should the image be configurable with an env var if we want to optionally fetch it from somewhere other than the default?

Reply:
Also, should the containers have names so we know what they were used for?